lib/active_model/serializer/adapter.rb

```diff
@@ -94,6 +95,7 @@ def include_meta(json)
         json[meta_key] = meta if meta
         json
       end
+      ##### AbstractAdapter
```

Comment:
I think you can remove and create a new issue for that, because you are right and we need to keep track of it in a more visible way :smile: